package.json

```diff
@@ -88,6 +88,7 @@
     "patternfly-react": "^2.19.1",
     "prop-types": "^15.6.0",
     "react": "^16.4.0",
+    "react-ace": "^6.2.0",
```

Comment:
@ekohl I've tried building this using npm2rpm version 3.1.0 but i keep getting this error: 
```
Found 5 dependencies - using bundle strategy
Making directory...FINISHED
Creating specs and downloading sources...---- npm2rpm ----
-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
 - Starting npm module download: https://registry.npmjs.org/react-ace/-/react-ace-6.2.0.tgz
 - Unpacking in /tmp/npm2rpm-17814Icc4KP55y8Ru ...
 - Finished extracting for react-ace
 - Reading package.json for react-ace
Warning: No README data
 - Finished reading package.json for react-ace
 - Fetching flattened list of production dependencies for react-ace
Callback must be a function
FINISHED
Copying specs...cp: cannot stat 'npm2rpm/SPECS/*': No such file or directory
```